metadata-service/plugin/src/main/java/com/datahub/plugins/configuration/ConfigProvider.java

```diff
@@ -0,0 +1,45 @@
+package com.datahub.plugins.configuration;
+
+import com.datahub.plugins.common.YamlMapper;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
+import lombok.extern.slf4j.Slf4j;
+
+
+@Slf4j
+public class ConfigProvider {
+  public static final String CONFIG_FILE_NAME = "config.yml";
+
+  /**
+   * Yaml file path of plugin configuration file. Content of this file should match with {@link Config}
+   */
+  private final Path configFilePath;
+  /**
+   *
+   */
+  private final Path _pluginBaseDir;
```

Comment:
Keep the names of these variables consistent. Either no underscore, or underscore.

Reply:
removed the underscore